demos/asp-net-core/Divergent.Sales.API.Host/Controllers/OrdersController.cs

```diff
@@ -12,40 +12,36 @@ public class OrdersController : ApiController
         [HttpGet, Route("{orderNumber}")]
         public dynamic Get(int orderNumber)
```

Comment:
if this PR is meant to refactor hosts this file should not be changed.

Reply:
"Host" as in the "API host" 😉